training_toolbox/tests/ssd_detector/__init__.py

```diff
@@ -0,0 +1,13 @@
+import importlib
+import os
+
+from .utils import download_test_data
```

Comment:
It is bad style to use dots. Is it possible to remove dots?

Reply:
I think it can stay here, its a test
dot used because there is a name conflict (another way is to using test_utils.py but the it looks like a test file) => the second major developer problem -- choose a variable/file name

Reply:
So the problem is in the name of one file -> `utils.py`. I can suggest a simple alternative - `general.py`. I would suggest renaming either `training_toolbox/tests/ssd_detector/utils.py` or `training_toolbox/utils.py`.

Reply:
test' utils.py to misc.py was renamed